app/controllers/order_details_controller.rb

```diff
@@ -72,6 +72,27 @@ def dispute
 
   private
 
+  def cancellation_success_path(order_detail)
+    if operator_cancellation?(order_detail)
+      facility_user_reservations_path(order_detail.facility, order_detail.user)
+    else
+      reservations_path
+    end
+  end
+
+  def cancel_with_fee(order_detail)
+    if operator_cancellation?(order_detail)
+      # When this is an operator who is not the ordering user; admin cancel with fee:
+      order_detail.cancel_reservation(session_user, OrderStatus.canceled_status, true, true)
```

Comment:
Maybe switch to using params here? The boolean values you're passing in are a bit confusing,  so doing ```admin_cancellation: true, admin_with_cancel_fee: true``` instead.